wwwapp/models.py

```diff
@@ -281,3 +289,23 @@ def result_in_percent(self):
 
     class Meta:
         unique_together = [('workshop', 'participant')]
+
+
+class ResourceYearPermission(models.Model):
+    display_name = models.CharField(max_length=32, blank=True)
+    # URL for menu entry, not displayed if empty
+    access_url = models.URLField(blank=True)
+    path = models.CharField(max_length=256, null=False, blank=False)
+    year = models.IntegerField(null=False, blank=False)
+
+    def __str__(self):
+        return "{} - {}".format(self.year, self.path)
+
+    def clean(self):
+        super().clean()
+        if self.access_url != "" and self.display_name == "":
+            raise ValidationError("Wyświetlana nazwa musi być ustawiona jeśli "
+                                  "URL dostępu jest ustawiony")
+
+    class Meta:
```

Comment:
Specifying default ordering here may be a good idea
![image](https://user-images.githubusercontent.com/1517255/61622406-fea63500-ac74-11e9-9f3d-3a7eb9e66d97.png)

Reply:
Maybe year then name?

Reply:
Sounds good